praas-app-ionic/src/components/Conduit/options.ts

```diff
@@ -0,0 +1,5 @@
+export const conduitTypes = [
```

Comment:
These could be moved into a constants file. But can be addressed in a separate ticket